src/components/NavBar/ProfilePicture.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import React, { useState } from "react";
+import Box from "@mui/material/Box";
+import Tooltip from "@mui/material/Tooltip";
+import IconButton from "@mui/material/IconButton";
+import Avatar from "@mui/material/Avatar";
+import Menu from "@mui/material/Menu";
+import MenuItem from "@mui/material/MenuItem";
+
+import { useRouter } from "next/navigation";
+import styled from "styled-components";
+
+const ProfileSettingsMenu = styled(Menu)`
+    margin-top: 50px;
+`;
+
+const ProfilePicture: React.FC<{}> = () => {
+    const router = useRouter();
+
+    const [profileMenuPosition, setProfileMenuPosition] = useState<HTMLElement | null>(null);
+    const handleOpenUserMenu = (event: React.MouseEvent<HTMLElement>): void => {
+        setProfileMenuPosition(event.currentTarget);
+    };
+    const handleCloseUserMenu = (): void => {
+        setProfileMenuPosition(null);
+    };
+
+    const handleSignOutClick = (): void => {
+        handleCloseUserMenu();
+
+        fetch("/auth/signout", {
```

Comment:
is any of this still needed with a `handleSignOutClick` apparently in `NavigationBar`?

Reply:
Profile Picture File actually isn't used at all I forgot it was there tbh. We kept it initially in case we wanted to switch back, but would you rather it deleted as it doesn't necessarily look like we will be doing so?